spec/base_decorator_spec.rb

```diff
@@ -0,0 +1,31 @@
+require_relative '../capitalize_decorator'
+require_relative '../trimmer_decorator'
+require_relative '../person'
+
+describe 'Decorators' do
+  before :all do
+    @person = Person.new(22, 'maximilianus')
+  end
+
+  describe '#capitalize_decorator' do
+    it 'returns a capitalized name' do
+      capitalized_person = CapitalizeDecorator.new(@person)
+      expect(capitalized_person.correct_name).to eq 'Maximilianus'
+    end
+  end
+
+  describe '#trimmer_decorator' do
+    it 'returns a name trimmed to 10 characters long if was initially longer' do
+      trimmed_person = TrimmerDecorator.new(@person)
+      expect(trimmed_person.correct_name).to eq 'maximilian'
+    end
+  end
+
+  describe '#trimmer_decorator' do
+    it 'returns a name trimmed to 10 characters long if was initially longer' do
+      capitalized_person = CapitalizeDecorator.new(@person)
+      trimmed_person = TrimmerDecorator.new(capitalized_person)
+      expect(trimmed_person.correct_name).to eq 'Maximilian'
+    end
```

Comment:
- Good job done so far, but according to the project requirements, all classes should have their own file i.e. `There should be one file per class`. Please test for the `TrimmerDecorator` and the `CapitalizedDecorator` in different files. Please fix this 😄